uw_sws/enrollment.py

```diff
@@ -151,23 +151,13 @@ def is_unfinished_pce_course(registration):
         is_src_location_pce(registration, REGISTRATION_SOURCE_PCE)
 
 
-def _json_to_unfinished_pce_course(json_data, aterm):
-    unf_pce_course = UnfinishedPceCourse()
-    unf_pce_course.section_ref = SectionReference(
-        term=aterm,
-        curriculum_abbr=json_data['Section']['CurriculumAbbreviation'],
-        course_number=json_data['Section']['CourseNumber'],
-        section_id=json_data['Section']['SectionID'],
-        url=json_data['Section']['Href']
-        )
-    unf_pce_course.start_date = parse(json_data['StartDate']).date()
-    unf_pce_course.end_date = parse(json_data['EndDate']).date()
-    unf_pce_course.feebase_type = json_data['FeeBaseType']
-    unf_pce_course.independent_start = json_data['IsIndependentStart']
-    unf_pce_course.is_credit = json_data['IsCredit']
-    unf_pce_course.request_status = json_data['RequestStatus']
-    unf_pce_course.meta_data = json_data['Metadata']
-    return unf_pce_course
+def _json_to_section_ref(reg_json, term):
+    return SectionReference(
+        term=term,
+        curriculum_abbr=reg_json['Section']['CurriculumAbbreviation'],
+        course_number=reg_json['Section']['CourseNumber'],
+        section_id=reg_json['Section']['SectionID'],
+        url=reg_json['Section']['Href'])
```

Comment:
Why aren't we using the UnfinishedPceCourse model here anymore?

Reply:
The original data in the enrollments search response is Registrations. 
Now all the data and methods are available in the Registration model, this one should be eliminated.